storage/engine/mvcc.go

```diff
@@ -1182,7 +1182,8 @@ func MVCCResolveWriteIntent(engine Engine, ms *proto.MVCCStats, key proto.Key, t
 		}
 		// Get the bytes for the next version so we have size for stat counts.
 		value := proto.MVCCValue{}
-		ok, _, valueSize, err := engine.GetProto(kvs[0].Key, &value)
+		var valueSize int64
+		ok, _, valueSize, err = engine.GetProto(kvs[0].Key, &value)
```

Comment:
Questioning the value of this change and the one at lines 1541-1544. What rationale is go-nyet giving?